cleverpush/src/main/java/com/cleverpush/CleverPush.java

```diff
@@ -3946,4 +3946,19 @@ private boolean shouldAutoRequestNotificationPermission() {
   public void setAutoRequestNotificationPermission(boolean autoRequestNotificationPermission) {
     this.autoRequestNotificationPermission = autoRequestNotificationPermission;
   }
+
+  public int getBadgeCount() {
```

Comment:
Can't we just call this method here dynamically instead of storing the badge number in SharedPreferences?
https://github.com/cleverpush/cleverpush-android-sdk/pull/198/files#diff-f38405a891ec6744c261383518105da501f20050da575b463715e8c2e7f0cf11R41